src/docs/cookbook/plugins/picture-using-camera.md

```diff
@@ -339,23 +337,19 @@ class TakePictureScreenState extends State<TakePictureScreen> {
             // Ensure that the camera is initialized.
             await _initializeControllerFuture;
 
-            // Construct the path where the image should be saved using the
-            // pattern package.
-            final path = join(
-              // Store the picture in the temp directory.
-              // Find the temp directory using the `path_provider` plugin.
-              (await getTemporaryDirectory()).path,
-              '${DateTime.now()}.png',
-            );
-
-            // Attempt to take a picture and log where it's been saved.
-            await _controller.takePicture(path);
+            // Attempt to take a picture and get the file `image` 
```

Comment:
Looks like there's some whitespace at the end of this line that's causing our formatting test to fail. Just need to nip it out.

Reply:
```suggestion
            // Attempt to take a picture and get the file `image`
```